tracex_project/extraction/logic/modules/module_activity_labeler.py

```diff
@@ -76,14 +76,12 @@ def __extract_activities(patient_journey_numbered, condition):
             messages.append(
                 {
                     "role": "user",
-                    "content": patient_journey_numbered
-                    + "\n\nConsider all important points regarding the course of the disease of "
-                    + condition,
+                    "content": f"Focus on those events that are related to the course of the disease of {condition}.\n"
+                    f"{patient_journey_numbered}",
```

Comment:
I would like to see minimum one of the few shot examples to feature this line ahead of the numbered pj. Also maybe give it two linebraks for a more visible border between the extra prompt and the input

Reply:
I'm afraid the OpenAI API and ChatGPT ignore line breaks. From what I've quickly found on the web is that you can preserve linebreaks by either adding `\n` to your string or using `"""` quotation marks that preserve line breaks.

Reply:
I added the linebreaks as suggested and also updated the prompt. Two of the few-shot examples now feature the additional line before the pj.